### ISSUE TITLE
Option to configure multiple RateDialog's on one project

### ISSUE THREAD

**Issue by jrcacd** [marcoscgdev/RateDialog#2]
I use this library in several projects.

In one of them, I needed to show different messages, so I ask if it is possible to show different RateDialogs in the same project, or if the library may have this functionality in the near future?

**Comment by marcoscgdev** [comment]
You can show the rate dialog directly by calling:
`RateDialog.show(this);`

And you can also show the rate dialog given a specific time (days or launches) by calling:
`RateDialog.with(this, 2, 0); // daysUntilPrompt, launchesUntilPrompt`

Are you asking for this, or do you want to create independent instances of the rate dialog (show the rate dialogs in different time for example)?

**Comment by jrcacd** [comment]
Is the second option

**Comment by marcoscgdev** [comment]
ok, I will add support for multiple dialog instances. I will close this issue when I'm done. Thank you for reporting it.

**Comment by marcoscgdev** [comment]
Fixed in [version 1.0.6](https://github.com/marcoscgdev/RateDialog/releases/tag/1.0.6).

### Creating custom dialog instance

```java
RateDialog rateDialog = new RateDialog(this, "custom_rate_dialog");
rateDialog.init();
```

You can also use custom config

```java
RateDialog rateDialog = new RateDialog(this, "custom_rate_dialog", daysUntilPrompt, launchesUntilPrompt);
```

Show the custom dialog instantly:

```java
RateDialog rateDialog = new RateDialog(this, "custom_rate_dialog");
rateDialog.showDialog();
```

NOTE: Use a different dialog key for each dialog instance.

**Comment by jrcacd** [comment]
Many thanks!
